src/transport/unicast/lease.c

```diff
@@ -28,6 +29,25 @@ int8_t _zp_unicast_send_keep_alive(_z_transport_unicast_t *ztu) {
     return ret;
 }
 
+int _zp_unicast_start_lease_task(_z_transport_t *zt, _z_task_attr_t *attr, _z_task_t *task) {
```

Comment:
As convention, return `int8_t` instead.
This goes back to a previous discussion on having a list of return codes.